docker/supabase/volumes/db/init/04-telescope-schema.sql

```diff
@@ -72,3 +80,9 @@ CREATE POLICY feeds_read_policy ON feeds
     FOR SELECT
     USING (true);
 
+ALTER TABLE build_logs ENABLE ROW LEVEL SECURITY;
+
+CREATE POLICY build_logs_policy ON build_logs
+    FOR ALL
+    USING (true);
+
```

Comment:
How to allow read/write for client with appropriate keys?

Reply:
@DukeManh